cpp/inc/bond/protocol/simple_binary.h

```diff
@@ -36,8 +36,8 @@
                      string, wstring        | count | characters |
                                             '-------'------------'
 
-                           count            variable encoded uint32 count of 1-byte (for
-                                            string) or 2-byte (for wstring) Unicode code
+                           count            uint32 count of 1-byte (for string)
+                                            or 2-byte (for wstring) Unicode code
                                             units (variable encoded in v2)
```

Comment:
Should this be updated in other places? For example, SimpleBinary.cs in the .NET version has the same comment. 

Also, what is the process to update the C++ API reference pages?